source/core-developers/coep-interceptor.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: COEP Interceptor
+parent:
+    title: Interceptors
+    url: interceptors.html
+---
+
+# Fetch Metadata Interceptor
+
+## Description
+
+Interceptor that implements Cross-Origin Embedder Policy on incoming requests.
+
+COEP prevents a document from loading any non-same-origin resources which don't explicitly grant the document permission to be loaded. This permission can be given via CORP (with a `Cross-Origin-Resource-Policy` header with a value which includes the requester, e.g `same-site`, or `cross-origin` for public resources)  or with CORS (by responding with appropriate `Access-Control-Allow-*` headers for requests in CORS mode).  The only allowed value for COEP is `require-corp`.
+COEP prevents the document from loading any framed documents which don’t opt-in by setting the COEP header. (`Cross-Origin-Embedder-Policy: require-corp`). This provides protection for documents that don’t restrict framing. A document that doesn’t set COEP cannot be framed by another document with COEP. All descendents of a document with COEP will also enforce the same restrictions.
```

Comment:
This paragraph mixes two different types of quotes! See: **’** vs **'** . Let's stick to the latter, please!

Reply:
Was debating between the two and forgot to keep only one

Reply:
Wow that was super nit. Great catch!